Sources/AblyChat/Messages.swift

```diff
@@ -1,10 +1,35 @@
 import Ably
 
+public protocol MessageSubscriptionResponse: Sendable {
```

Comment:
We already have `MessageSubscription`; what is this new type for?

Reply:
using this now